pkg/credential/credential.go

```diff
@@ -18,16 +18,17 @@ const (
 	clientSecretKey   = "azure.azureoperator.clientsecret"
 	defaultAzureGUID  = "37f13270-5c7a-56ff-9211-8426baaeaabd"
 	partnerIDKey      = "azure.azureoperator.partnerid"
+	singleTenantLabel = "giantswarm.io/single-tenant-service-principal"
```

Comment:
Minor nit: Maybe move this to `label` package?

Reply:
Done